docs/src/tutorials/tutorial_3.md

```diff
@@ -54,13 +54,10 @@ You have now successfully downloaded MimiDICE2010 to your local machine.
 
 The next step is to run DICE using the provided API for the package:
 
-```jldoctest tutorial2; output = false, filter = r".*"s
+```julai
```

Comment:
"julia"?

Reply:
oh shoot yes thanks, let me make sure the tests all pass then I'll ping you again